frontend/app/models/recurrence-match-positions.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default [
+  Ember.Object.create({ value: 0, label: "First" }),
+  Ember.Object.create({ value: 1, label: "Second" }),
+  Ember.Object.create({ value: 2, label: "Third" }),
+  Ember.Object.create({ value: 3, label: "Fourth" }),
+  Ember.Object.create({ value: 4, label: "Last" })
```

Comment:
Equivalent here, we are using in the server recurrence_match `-1 => "Last"`

I would also include a `-2 => "Next to last"` option